### DISCUSSION TITLE
SSB signal receiver issues

### DISCUSSION THREAD

**Discussion by ABD9588** [egzumer/uv-k5-firmware-custom#141] [Q&A]
Hi, Bug report, I have two UV-K6 that are also installed with firmware 0.17. I found that every time switch to SSB, they will not auto receive. After pressing the monitor mode on/off button, they will start to receive normally in SSB.

**Comment by ErikS-web** [comment]
How did you test ?
What were you listening to ?

**Reply by ABD9588** [reply]
Dear  ErikS-web

Thank you for your prompt reply. Maybe I operated the radio incorrectly. The problem has been solved.

Thx
